src/main/java/com/sap/oss/phosphor/fosstars/model/RatingRepository.java

```diff
@@ -116,6 +120,31 @@ private static OssRulesOfPlayRating ossRulesOfPlayRating() {
     return new OssRulesOfPlayRating();
   }
 
+  /**
+   * Loads a {@link OssArtifactSecurityRating}.
+   *
+   * @return An instance of {@link OssArtifactSecurityRating}.
+   */
+  private static OssArtifactSecurityRating ossArtifactSecurityRating() throws IOException {
+    OssArtifactSecurityScore ossArtifactSecurityScore = new OssArtifactSecurityScore();
+
+    ArtifactVersionSecurityScore artifactVersionSecurityScore =
+        ossArtifactSecurityScore.getArtifactVersionSecurityScore();
+    artifactVersionSecurityScore.weights().update(
+        loadScoreWeights("com/sap/oss/phosphor/fosstars/model/score/oss/"
+            + "OssArtifactSecurityScoreWeights.json"));
+
+    OssSecurityScore ossSecurityScore = ossArtifactSecurityScore.getOssSecurityScore();
+    ossSecurityScore.weights().update(
+        loadScoreWeights("com/sap/oss/phosphor/fosstars/model/score/oss/"
+            + "OssSecurityScoreWeights.json"));
```

Comment:
Please don't create a new instance of `OssSecurityScore` since we already have it. Please use the existing one to initialize `OssArtifactSecurityScore` (you can take from the `OssSecurityRating` offered by the rating repository).

`OssArtifactSecurityScore`'s constructor should take sub-scores as parameters.

Reply:
Please see https://github.com/SAP/fosstars-rating-core/pull/423#discussion_r603434186

Feel free to open an issue for this. I'll take it then.